api/client/src/main/java/org/projectnessie/client/auth/oauth2/AuthorizationCodeFlow.java

```diff
@@ -141,7 +144,7 @@ private void abort() {
     redirectUriFuture.cancel(true);
   }
 
-  public Tokens fetchNewTokens() {
+  public Tokens fetchNewTokens(@Nullable Tokens ignored) {
```

Comment:
> nit: `@Override`

👍 

> Should we assert that the `Tokens` parameter is in fact `null` for the sake of correctness?

Not sure, the `OAuth2Client` *could* invoke these flows with whatever tokens are currently available. That wouldn't be incorrect per se imho.

> Alternatively, maybe we could have two methods: `fetchNewTokens()` and `refreshTokens(Tokens)` with the base class throwing `UnsupportedOperationException` by default?

Again not sure, because only one of the two methods would work, the other would necessarily throw.